specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2018-08-01-preview/costmanagement.json

```diff
@@ -2206,8 +2336,8 @@
         "tags": [
           "Alerts"
         ],
-        "operationId": "Alerts_GetByEnrollment",
-        "description": "Gets the alert for an enrollment by alert ID.",
+        "operationId": "Alerts_GetByBillingAccount",
```

Comment:
This is a breaking change. Could you please explain why this change was made in an existing version?

Reply:
@saschein Please respond.

Reply:
Sorry,
We are still in private-preview and are currently only using this api-version, we wanted to prevent adding a new one before we stabilize this version.
If this change must be in a new api-version we'll discard it and create a new PR later.

Thanks

Reply:
Yes. Normally, we do not encourage breaking changes. It would be advisable to do the changes in new version

Reply:
@saschein As mentioned in the previous comment, I am closing this PR. Please open a .new one with correct version